### PR TITLE
Four horsemen of design.

### PR DESCRIPTION
**Pull request by teh**

### CODE REVIEW

File: content/articles/four-horsemen-of-design.md [hunk 1]
```diff
@@ -0,0 +1,281 @@
...
+  <div class="fhd-card small" style="top: 20px;width: 50px;;"></div>
+  <div class="fhd-card small" style="top: 20px;width: 50px;;left: 100px;;"></div>
+  <div class="fhd-card small" style="top: 40px;;width: 50px;;"></div>
+  <div class="fhd-card small" style="top: 40px;;width: 50px;;left: 100px;;"></div>
+  <div class="fhd-card small" style="top: 60px;width: 50px;;"></div>
+  <div class="fhd-card small" style="top: 60px;width: 50px;;left: 100px;;"></div>
+  <div class="fhd-card small" style="top: 80px;width: 50px;;"></div>
+  <div class="fhd-card small" style="top: 80px;width: 50px;;left: 100px;;"></div>
+  <div class="fhd-card small" style="top: 100px;width: 50px;;"></div>
+  <div class="fhd-card small" style="top: 100px;width: 50px;;left: 100px;;"></div>
```
Comment (Keats) [inline content/articles/four-horsemen-of-design.md]: too many ;; in that html example

**Comment by teh** [comment]
Ffixed the `;;` issue. I'll give it another read later this afternoon and then publish tomorrow?

**Comment by Keats** [comment]
i'd want barbara to read it but we're flying very soon so won't have the
time right now

2015-02-27 12:44 GMT+00:00 teh <notifications@github.com>:

**Comment by teh** [comment]
cool, no hurry, lets publish some time next week

**Comment by teh** [comment]
I'm super sorry I published this by accident when moving the server. Now that it is live I'm merging because people with feed readers will have seen it pop up already.

**Pull request merged**